src/diffusers/loaders.py

```diff
@@ -839,6 +839,9 @@ def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Di
         weight_name = kwargs.pop("weight_name", None)
         use_safetensors = kwargs.pop("use_safetensors", None)
 
+        # set lora scale to a reasonable default
+        self._scale = 1
```

Comment:
But we'd want the users to also specify this explicitly if they want to, no? 

For that, don't you think exposing an argument for it makes sense?

Reply:
Yes, actually users should never set them argument themselves (which is why it's marked private with a `_`). This should just be used to make it work with LoRA

Reply:
I think we have a misunderstanding here, let's maybe take this offline in a quick call next week :-)